client/src/Components/QuestionsAndAnswers/QnaIndex.jsx

```diff
@@ -65,15 +66,32 @@ var QnaIndex = (props) => {
         setDefaultQ(response.data);
       })
       .catch((error) => {
-        console.log('error within getting data after posting from client')
+        console.log('error within getting data after posting answer from client')
       })
     })
     .catch((error) => {
-      console.log('error within posting data from client')
+      console.log('error within posting answer from client')
     })
 
   };
 
+  var postQuestion = (body) => {
+    axios.post('/snuggie/post/question', body)
+    .then(() => {
+      axios.get('/snuggie/qa/questions', {params : {product_id: 40713, count: 100}})
+      .then((response) => {
+        setQuestion(response.data);
+        setDefaultQ(response.data);
+      })
+      .catch((error) => {
+        console.log('error within getting data after posting question from client')
+      })
+    })
+    .catch((error) => {
+      console.log('error within posting question from client side')
+    })
+  };
```

Comment:
Nested promises is bad practice (although not too bad here because it's just one nest). If you want, you could just chain it like 
```
axios.post()
  .catch( // error posting)
  .then(() => {
    return axios.get()
  })
  .then((response) => {
    // set stuff
  })
  .catch( // error getting)
```

Of course, it is up to you.

Reply:
Followed through with the recommendation. Thank you